src/components/Player/Player.jsx

```diff
@@ -49,35 +54,38 @@ const getPlayerSubroute = (info, playerId) => {
   }
 };
 
-const Player = ({ playerId, info, params }) => {
-  if (!playerId) {
+const Player = ({ params: { accountId, info, subInfo } }) => {
```

Comment:
this is destructuring syntax for destructuring multiple levels (in this case, props.params.WhatIWant). It seems like only the component being given to a `<Route component={MyComponent} />` seems to be given the params as props. I haven't found a HOC that will give them to components yet. I'm sure I could make one and store the stuff in redux but it seems unnecessary for now since I'm only passing a couple levels deep. If I have to pass more levels deep then I will probably do that.